python/nx-cugraph/nx_cugraph/scripts/print_table.py

```diff
@@ -59,7 +59,7 @@ def main(path_to_info=None, *, file=sys.stdout):
     if path_to_info is None:
         path_to_info = get_path_to_info(version_added_sep=".")
     lines = ["networkx_path,dispatch_name,version_added,plc,is_incomplete,is_different"]
-    lines.extend(",".join(info) for info in path_to_info.values())
+    lines.extend(",".join([str(i) for i in info]) for info in path_to_info.values())
```

Comment:
This is indeed the fix I mentioned, although I prefer the solution here: https://github.com/rapidsai/cugraph/pull/4108/files#diff-c0145493e321fbd8f77cf9de0c1b0889eeceaebe06c780511cc5b13646785489R62

```suggestion
    lines.extend(",".join(map(str, info)) for info in path_to_info.values())
```